src/db/migrations/20240822032133_subscriber_email_preferences.js

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export async function up(knex) {
+  await knex.schema
+    .createTable('subscriber_email_preferences', function(table) {
+      table.increments('id').primary();
+      table.integer("subscriber_id").references("subscribers.id").notNullable().onDelete('CASCADE').onUpdate('CASCADE');
```

Comment:
These always make me nervous 😅 I just double-checked and this should delete the row in `subscriber_email_preferences` when a row in `subscribers` is deleted, but not vice versa, that's the intent here correct? :)

Reply:
Correct. The cascade is one way